tools/common/compilerconfig.flow

```diff
@@ -241,7 +241,7 @@ getCompilerConfig() {
                 beautify,
                 verbose,
                 nwjsmode,
-                if (fc == "") "resources/fontconfig.json" else fc,
+                if (fc == "") flowDir + "resources/fontconfig.json" else fc,
```

Comment:
I think you should first use "fileExists" on the relative path, and if that exists, use that. If that does not exist, then OK to use the flowDir path. We do want to support relative paths.